java/src/jmri/jmrix/AbstractThrottleManager.java

```diff
@@ -416,130 +322,6 @@ public void cancelThrottleRequest(LocoAddress la, ThrottleListener l) {
-     *
-     */
-    @Deprecated
-    @Override
-    public void stealThrottleRequest(BasicRosterEntry re, ThrottleListener l,boolean steal){
-        if (steal) {
-            responseThrottleDecision(re.getDccLocoAddress(), l, ThrottleListener.DecisionType.STEAL);
-        }
-        else {
-            cancelThrottleRequest(re.getDccLocoAddress(), l);
-        }
-    }
-    
-    /**
-     * Steal a requested throttle.
-     * <p>
-     * This is a convenience version of the call, which uses system-specific
-     * logic to tell whether the address is a short or long form.
-     *
-     * @deprecated since 4.15.7; use #responseThrottleDecision
-     *
-     */
-    @Deprecated
-    @Override
-    public void stealThrottleRequest(int address, ThrottleListener l,boolean steal){
-        boolean isLong = true;
-        if (canBeShortAddress(address)) {
-            isLong = false;
-        }
-        DccLocoAddress la = new DccLocoAddress(address, isLong);
-        if (steal) {
-            responseThrottleDecision(la, l, ThrottleListener.DecisionType.STEAL);
-        }
-        else {
-            cancelThrottleRequest(la, l);
-        }
-    }
-
-    /**
-     * Steal a requested throttle.
-     *
-     * @deprecated since 4.15.7; use #responseThrottleDecision
-     *
-     * @param address desired decoder address
-     * @param isLong  true if requesting a DCC long (extended) address
-     * @param l  ThrottleListener requesting the throttle steal occur.
-     * @param steal true if the request should continue, false otherwise.
-     * @since 4.9.2
-     */
-    @Deprecated
-    @Override
-    public void stealThrottleRequest(int address, boolean isLong, ThrottleListener l,boolean steal){
-        DccLocoAddress la = new DccLocoAddress(address, isLong);
-        if (steal) {
-            responseThrottleDecision(la, l, ThrottleListener.DecisionType.STEAL);
-        }
-        else {
-            cancelThrottleRequest(la, l);
-        }
-    }
-    
-    /**
-     * @deprecated since 4.15.7; use #responseThrottleDecision
-     */
-    @Deprecated
-    @Override
-    public void stealThrottleRequest(LocoAddress address, ThrottleListener l, boolean steal){
-        if (steal) {
-            responseThrottleDecision(address, l, ThrottleListener.DecisionType.STEAL);
-        }
-        else {
-            cancelThrottleRequest(address, l);
-        }
-    }
-    
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public void responseThrottleDecision(int address, ThrottleListener l, ThrottleListener.DecisionType decision) {
-        boolean isLong = true;
-        if (canBeShortAddress(address)) {
-            isLong = false;
-        }
-        responseThrottleDecision(address, isLong, l, decision);
-
-    }
-    
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public void responseThrottleDecision(int address, boolean isLong, ThrottleListener l, ThrottleListener.DecisionType decision) {
```

Comment:
This is likely a convenience for scripts, so I don't think it should be removed.